packages/cadl-ranch-specs/http/azure/resource-manager/models/resources/mockapi.ts

```diff
@@ -42,6 +42,144 @@ const validNestedResource = {
   },
 };
 
+const validExtensionResource = {
+  id: `/subscriptions/${SUBSCRIPTION_ID_EXPECTED}/resourceGroups/${RESOURCE_GROUP_EXPECTED}/providers/Azure.ResourceManager.Models.Resources/roleAssignments/role`,
+  name: "role",
+  type: "extensionResource",
```

Comment:
type is wrong

check https://github.com/Azure/cadl-ranch/blob/main/packages/cadl-ranch-specs/http/azure/resource-manager/models/resources/mockapi.ts#L11

Reply:
Fixed in the new version.